style.css

```diff
@@ -17,4 +17,18 @@ body {
     padding-block: 1em;
     border-top: 1px solid #007bff;
     border-bottom: 3px solid orangered;
+}
+
+.power-switch button{
+    border-width: 0.2px;
+    border-radius: 8px;
+    cursor: pointer;
+    background-color: rgb(219, 191, 180);
+    color: black;
+    
+}
+
+.power-switch button:hover{
+    background-color: grey;
+    color: white;
```

Comment:
Im going to request that you create an issue regarding this PR so that changes can be correctly discussed, there is no need to merge the PR for the sake of it, let's get great changes in place!